torch/onnx/symbolic_opset11.py

```diff
@@ -48,5 +48,27 @@ def unique_dim(g, self, dim, sorted, return_inverse, return_counts):
     return u, inverse_indices, counts
 
 
+@parse_args('v', 'v', 'i', 'i', 'i', 'none')
+def topk(g, self, k, dim, largest, sorted, out=None):
```

Comment:
the code in this function is very similar to the one is symbolic_opset10.py. 
Would it be possible to refactor to minimise code duplication?

Reply:
Yeah that's right, fixed it

Reply:
My comment above is related to this. To clarify, I agree that we should not duplicate code, therefore I am OK with the code as it is. But for code readability it would be better that each opset version is written in the respective file with only the common portions of code captured outside.